### PR TITLE
Quartz ore handling

### PR DESCRIPTION
**Pull request by jammet**
I've never done this, I don't understand what I'm doing, but if copypasta serves right, there is a chance that this is correct.
I'm not putting CLAY in at the same time because, see above, I don't know what I'm doing. ;)

### CODE REVIEW

File: src/Blocks/BlockOre.h [hunk 1]
```diff
@@ -50,6 +50,10 @@ class cBlockOreHandler :
 				a_Pickups.push_back(cItem(E_ITEM_COAL));
 				break;
 			}
+			case E_BLOCK_QUARTZ_ORE:
+				a_Pickups.push_back(cItem(E_ITEM_QUARTZ));
```
Comment (NiLSPACE) [inline src/Blocks/BlockOre.h]: You forgot to place a brace here
```cpp
case XYZ:
{
   // Add pickup
}
```

**Comment by NiLSPACE** [comment]
Does quartz ore give one quartz or more?

**Comment by jammet** [comment]
Nether quartz ore <http://minecraft.gamepedia.com/Nether_quartz_ore> mined
using a pickaxe <http://minecraft.gamepedia.com/Pickaxe> will drop one unit
of nether quartz. If the pickaxe is enchanted with fortune, it may drop an
extra unit per level of fortune, up to a maximum of 4 with Fortune III. If
the pickaxe is enchanted with silk touch, the ore will be dropped.

**Comment by jammet** [comment]
About the brace, want me to try this again?

**Comment by NiLSPACE** [comment]
You can just add it in this pull request.

**Comment by jammet** [comment]
Not sure if I need to click another different pull request button, but I made the change.

File: src/Blocks/BlockOre.h [hunk 2]
```diff
@@ -50,6 +50,11 @@ class cBlockOreHandler :
 				a_Pickups.push_back(cItem(E_ITEM_COAL));
 				break;
 			}
+			case E_BLOCK_QUARTZ_ORE:
+			{
+				a_Pickups.push_back(cItem(E_ITEM_QUARTZ));
+				break;
+		        }
```
Comment (NiLSPACE) [inline src/Blocks/BlockOre.h]: You're using a combination of spaces and tabs here. Could you change it to tabs?

**Comment by jammet** [comment]
Like so?

File: src/Blocks/BlockOre.h (hunk 2 above)
Comment (jammet) [inline src/Blocks/BlockOre.h]: I think I got them all. Is this the way you need it to be now?

File: src/Blocks/BlockOre.h [hunk 3]
```diff
@@ -50,6 +50,11 @@ class cBlockOreHandler :
 				a_Pickups.push_back(cItem(E_ITEM_COAL));
 				break;
 			}
+			case E_BLOCK_QUARTZ_ORE:
```
Comment (NiLSPACE) [inline src/Blocks/BlockOre.h]: Rename this to `E_BLOCK_NETHER_QUARTZ_ORE`

Comment (jammet) [inline src/Blocks/BlockOre.h]: Clearly. I'm the only man who could've done it. -cough-

Done that.

File: src/Blocks/BlockOre.h [hunk 4]
```diff
@@ -50,6 +50,11 @@ class cBlockOreHandler :
 				a_Pickups.push_back(cItem(E_ITEM_COAL));
 				break;
 			}
+			case E_BLOCK_NETHER_QUARTZ_ORE:
+			{
+				a_Pickups.push_back(cItem(E_ITEM_QUARTZ));
```
Comment (NiLSPACE) [inline src/Blocks/BlockOre.h]: Rename `E_ITEM_QUARTZ` to `E_ITEM_NETHER_QUARTZ`

**Comment by NiLSPACE** [comment]
Any chance you could do all these commits into a single commit?

**Comment by jammet** [comment]
Should I create a whole new one? I don't know how to otherwise "combine"
these.

**Comment by NiLSPACE** [comment]
I think @madmaxoft knows how to do that.

(Don't forget to rename `E_ITEM_QUARTZ` to `E_ITEM_NETHER_QUARTZ` btw)

**Comment by jammet** [comment]
Yes, I did rename it... or at least I think I did.
Sorry, in the end, you had a lot of time wasted just because I tried doing this.
Probably should stick to just pointing things out.

I'll wait for MadMaxoft.

**Comment by madmaxoft** [comment]
Normally you'd need to use an advanced git technique called "interactive rebasing", which is a bit involved. I'll see if I can do it for you (tomorrow).

**Comment by jammet** [comment]
Thank you! :)

**Comment by madmaxoft** [comment]
I managed to keep you as the author of the commit :)

**Comment by jammet** [comment]
O, the blood I shed, fighting for it to see the day ;).

Thanks! :)

**Pull request closed without merging**